core/web/job_specs_controller_test.go

```diff
@@ -333,7 +334,11 @@ func TestJobSpecsController_Create_FluxMonitor_enabled(t *testing.T) {
 	resp, cleanup := client.Post("/v2/specs", bytes.NewBuffer(jsonStr))
 	defer cleanup()
 
-	require.Equal(t, http.StatusText(http.StatusOK), http.StatusText(resp.StatusCode))
+	body, _ := ioutil.ReadAll(resp.Body)
```

Comment:
This seems unrelated?

Reply:
It's an improvement to the failure message. I think it's good.

Reply:
It might have been better as its own PR.

Its not completely unrelated though.  This error was probably triggered when updating the code, without enough explanation.

Reply:
Possibly use [cltest.AssertServerResponse](https://github.com/smartcontractkit/chainlink/blob/develop/core/internal/cltest/cltest.go)?